### ISSUE TITLE
mContext Null Error

### ISSUE THREAD

**Issue by sssvrock** [barteksc/AndroidPdfViewer#33]
Hi 

@barteksc Can you please help me on below (PFB)

E/com.shockwave.pdfium.PdfiumCore: mContext may be null
- I am not sure why it is coming and not able to load pdf in viewpager at this moment.

Thanks

**Comment by barteksc** [comment]
I cannot check what is wrong If there is no more info from logcat.

**Comment by sssvrock** [comment]
Hi,
  I am only getting this line in error message
 **E/com.shockwave.pdfium.PdfiumCore: mContext may be null **

In my case I am displaying the pdf files from local storage in view pager as a page by page display.

Thanks
Vinod

**Comment by ThePaulrBear** [comment]
I see the same error, and each time it is followed by the following:
```
08-26 15:34:55.013 com.pcmedical.controller W/System.err: java.lang.NullPointerException
08-26 15:34:55.013 com.pcmedical.controller W/System.err:     at com.shockwave.pdfium.PdfiumCore.renderPageBitmap(PdfiumCore.java:216)
08-26 15:34:55.013 com.pcmedical.controller W/System.err:     at com.github.barteksc.pdfviewer.RenderingAsyncTask.proceed(RenderingAsyncTask.java:123)
08-26 15:34:55.013 com.pcmedical.controller W/System.err:     at com.github.barteksc.pdfviewer.RenderingAsyncTask.doInBackground(RenderingAsyncTask.java:77)
08-26 15:34:55.013 com.pcmedical.controller W/System.err:     at com.github.barteksc.pdfviewer.RenderingAsyncTask.doInBackground(RenderingAsyncTask.java:34)
08-26 15:34:55.013 com.pcmedical.controller W/System.err:     at android.os.AsyncTask$2.call(AsyncTask.java:264)
08-26 15:34:55.013 com.pcmedical.controller W/System.err:     at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:305)
08-26 15:34:55.013 com.pcmedical.controller W/System.err:     at java.util.concurrent.FutureTask.run(FutureTask.java:137)
08-26 15:34:55.013 com.pcmedical.controller W/System.err:     at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1076)
08-26 15:34:55.013 com.pcmedical.controller W/System.err:     at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:569)
08-26 15:34:55.013 com.pcmedical.controller W/System.err:     at java.lang.Thread.run(Thread.java:856)
```

**Comment by barteksc** [comment]
Probably fixed in 2.0.2

**Comment by shihabmi7** [comment]
I am facing same issue now a days.
full logcat : 

`11-02 22:11:37.289 11531-12414/com.alhikmah.muslimbiye E/com.shockwave.pdfium.PdfiumCore: mContext may be null
11-02 22:11:37.289 11531-12414/com.alhikmah.muslimbiye W/System.err: java.lang.NullPointerException
11-02 22:11:37.289 11531-12414/com.alhikmah.muslimbiye W/System.err:     at com.shockwave.pdfium.PdfiumCore.renderPageBitmap(PdfiumCore.java:216)
11-02 22:11:37.289 11531-12414/com.alhikmah.muslimbiye W/System.err:     at com.github.barteksc.pdfviewer.RenderingAsyncTask.proceed(RenderingAsyncTask.java:123)
11-02 22:11:37.289 11531-12414/com.alhikmah.muslimbiye W/System.err:     at com.github.barteksc.pdfviewer.RenderingAsyncTask.doInBackground(RenderingAsyncTask.java:77)
11-02 22:11:37.289 11531-12414/com.alhikmah.muslimbiye W/System.err:     at com.github.barteksc.pdfviewer.RenderingAsyncTask.doInBackground(RenderingAsyncTask.java:34)
11-02 22:11:37.289 11531-12414/com.alhikmah.muslimbiye W/System.err:     at android.os.AsyncTask$2.call(AsyncTask.java:287)
11-02 22:11:37.294 11531-12414/com.alhikmah.muslimbiye W/System.err:     at java.util.concurrent.FutureTask.run(FutureTask.java:234)
11-02 22:11:37.294 11531-12414/com.alhikmah.muslimbiye W/System.err:     at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1080)
11-02 22:11:37.294 11531-12414/com.alhikmah.muslimbiye W/System.err:     at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:573)
11-02 22:11:37.294 11531-12414/com.alhikmah.muslimbiye W/System.err:     at java.lang.Thread.run(Thread.java:841)
11-02 22:11:37.294 11531-12414/com.alhikmah.muslimbiye A/libc: Fatal signal 11 (SIGSEGV) at 0x03040264 (code=1), thread 12414 (AsyncTask #3)`
